src/core/task.h

```diff
@@ -0,0 +1,51 @@
+#ifndef CORE_TASK_H
+#define CORE_TASK_H
+
+#include <stdint-gcc.h>
+/// status of task
+typedef enum{
+	TASK_ACTIVE, ///< task is currently running
+	TASK_ACTIVE_WTM,  ///< task is currently active but it is waiting on motion
+	TASK_READY, ///< task is ready
+	TASK_DONE, ///< finished task
+	TASK_BLOCKED ///< task is blocked currently blocked because of some obstacle
```

Comment:
The comment includes a typo.
Either `taks is blocked currently, because of some obstacle`,
or `task is currently blocked because of some obstacle`.